### DISCUSSION TITLE
How to use nerd fonts

### DISCUSSION THREAD

**Discussion by angelog0** [msys2/MINGW-packages#18195] [General]
I see you have added nerd fonts packages. I do not understand how they integrate with Windows.

Suppose I install

```
pacman -S mingw-w64-ucrt-x86_64-otf-droid-nerd
```

how this font is visible by Windows apps? How editors, browsers etc, could be configured to use those fonts?

I had in the past installed manually (at the Windows manner) nerd fonts downloading them from the site and installing them as administrator etc 

Could you give some clarifications?

TIA

**Comment by ognevny** [comment]
first of all, I've added nerd-fonts because they are needed for starship's theme prompts, so starship should find it (but I can't test it now)

> how this font is visible by Windows apps?

AFAIK the only way to make font visible is opening .ttf or .otf files and clicking 'install' in appeared window. fonts are located in `{mingw_prefix}/share/fonts/TTF` (`OTF`). maybe Windows has an utility to make directory as source for the fonts.

**Reply by angelog0** [reply]
Basically installing the nerd fonts in MSYS2 is equivalent to downloading them from the site, then we have to install them manually with the usual procedure. Thanks for clarifications..

**Reply by ognevny** [reply]
the only one difference is easier update :)

**Reply by angelog0** [reply]
Indeed.. Thanks
